
irl_robot_command: "movej"
irl_robot_accl: 3.15
irl_robot_vel: 3.15
irl_robot_com_time: 1
ur5:
  control_params:
    p_gain: 2
    i_gain: 1
    d_gain: 0.0
    m_max_velocity: 1.57
    m_i_max: 0.5
    m_i_min: -0.5
    m_max_accel: 3.14

# KDL params
kdl_urdf:  'custom_ur5.urdf' # assuming file is in urdf directory
# kdl_urdf:  'custom_ur5.urdf' # assuming file is in urdf directory
# kdl_urdf:  'kdl_custom_ur5_1.urdf' # assuming file is in urdf directory
# kdl_urdf:  'kdl_custom_ur5.urdf' # assuming file is in urdf directory
ik_goal:
  x     : 0.2   # meters
  y     : 0.4   # meters 
  z     : 0.35  # meters
  yaw   : 0.0   # degrees
  pitch : 90.0   # degrees
  roll  : 0.0   # degrees
z_offset : 0.0
gripper_sim : 0.0
gripper_real: 0
gripper_real_step: 8

home_joints: [115.55, -105.80, 99.89, -84.24, -89.98, 25.42]

# DS4 params
limit_cart : 0.8
cart_step  : 0.02


# experiment goals 1
goals:                
  - [0.30,  0.58, 0.25]
  - [0.30,  0.38, 0.25]
  - [0.30,  0.18, 0.25]
  # - [0.30, -0.02, 0.25]     
  # - [0.30, -0.22, 0.25]     
  # - [0.30, -0.40, 0.25]
  # - [0.46,  0.60, 0.25]
  # - [0.46,  0.40, 0.25]
  # - [0.46,  0.20, 0.25]     
  # - [0.46, -0.00, 0.25]  
  # - [0.46, -0.20, 0.25]
  # - [0.46, -0.40, 0.25]
  # - [0.64,  0.60, 0.25]
  # - [0.64,  0.40, 0.25]     
  # - [0.64,  0.20, 0.25]  
  # - [0.64, -0.00, 0.25]  
  # - [0.64, -0.20, 0.25]  
  - [0.64, -0.40, 0.25]




# experiment goals 2
# 0: [0.50, 0.0, 0.08],
# 1: [0.50, 0.2, 0.08],
# 2: [0.50, 0.4, 0.08],
# 3: [0.50,-0.2, 0.08],
# 4: [0.50,-0.4, 0.08],
# 5: [0.25, 0.0, 0.08],
# 6: [0.25, 0.2, 0.08],
# 7: [0.25, 0.4, 0.08],
# 8: [0.25,-0.2, 0.08],
# 9: [0.25,-0.4, 0.08]       

# experiment goals 3
# 0:  [0.6, -0.6, 0.08],
# 1:  [0.6, -0.4, 0.08],
# 2:  [0.6, -0.2, 0.08],
# 4:  [0.6,  0.0, 0.08],     
# 5:  [0.6,  0.2, 0.08],
# 6:  [0.6,  0.4, 0.08],
# 7:  [0.6,  0.6, 0.08],  
# 8:  [0.35,  -0.6, 0.08],
# 9:  [0.35,  -0.4, 0.08],
# 10: [0.35, -0.2, 0.08],
# 11: [0.35,  0.0, 0.08],  
# 12: [0.35,  0.2, 0.08],
# 13: [0.35,  0.4, 0.08],
# 14: [0.35,  0.6, 0.08]

# experiment goals 4
# 0:  [0.25,  0.25, 0.08],
# 1:  [0.25,  0.15, 0.08],
# 2:  [0.25,  0.05, 0.08],
# 3:  [0.25, -0.10, 0.08],     
# 4:  [0.25, -0.25, 0.08],     
# 5:  [0.45,  0.25, 0.08],
# 6:  [0.45,  0.15, 0.08],
# 7:  [0.45,  0.05, 0.08],
# 8:  [0.45, -0.10, 0.08],     
# 9:  [0.45, -0.25, 0.08],  
# 10: [0.60,  0.25, 0.08],
# 11: [0.60,  0.15, 0.08],
# 12: [0.60,  0.05, 0.08],
# 13: [0.60, -0.10, 0.08],     
# 14: [0.60, -0.25, 0.08]